tests/test_RestartSerialization.cpp

```diff
@@ -75,6 +81,28 @@ TEST_FOR_TYPE_NAMED_OBJ(ATE, AdaptiveTimeSteppingEbosPID, serializationTestObjec
 TEST_FOR_TYPE_NAMED_OBJ(ATE, AdaptiveTimeSteppingEbosPIDIt, serializationTestObjectPIDIt)
 TEST_FOR_TYPE_NAMED_OBJ(ATE, AdaptiveTimeSteppingEbosSimple, serializationTestObjectSimple)
 
+BOOST_AUTO_TEST_CASE(EclGenericVanguard)
+{
+    auto in_params = Opm::EclGenericVanguard::serializationTestParams();
+    Opm::EclGenericVanguard val1(std::move(in_params));
+    Opm::Serialization::MemPacker packer;
+    Opm::Serializer ser(packer);
+    ser.pack(val1);
+    size_t pos1 = ser.position();
```

Comment:
It probably doesn't matter a whole lot, but I think `pos1` (and `pos2` below) could be marked `const`.

Reply:
sure.